src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,15 @@
-export const CommentList = () => <>Put the list here</>;
+import React from 'react';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    <ul>
```

Comment:
Remember to remove default `list-style` when using list otherwise unwanted bullet points may show up:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/52002856/428e5527-ac3d-4f15-9159-88222144b804)

The base page provided does not use `ul`s and `li`s here, so you may just use the same tags as it